OWML.Patcher/VrPatcher.cs

```diff
@@ -1,129 +1,37 @@
-﻿using System;
+﻿using OWML.Common;
 using System.IO;
-using System.Linq;
-using System.Security.Cryptography;
-using System.Text;
-using BsDiff;
-using OWML.Common;
 
 namespace OWML.Patcher
 {
     public class VRPatcher
     {
         private readonly IOwmlConfig _owmlConfig;
         private readonly IModConsole _writer;
-        private readonly SHA256 _sha;
 
         private static readonly string[] PluginFilenames = { "openvr_api.dll", "OVRPlugin.dll" };
-        private static readonly string[] PatchChecksums =
-        {
-            "cacc71fcb141d936f1b59e57bf10dc52e8edb3481988379f7d95ecb65c4d3c90",
-            "d3979abb3b0d2468c3e03e2ee862d5297f5885bd9fc8f3b16cb16805e010d097",
-            "7ed2c835ec6653009d29b6b7fa9dc36cd754f64b2f359f7ca635ec6cd4ad8c32"
-        };
 
         public VRPatcher(IOwmlConfig owmlConfig, IModConsole writer)
         {
             _owmlConfig = owmlConfig;
             _writer = writer;
-            _sha = SHA256.Create();
         }
 
         public void PatchVR(bool enableVR)
         {
-            PatchGlobalManager(enableVR);
+            var binaryPatcher = new BinaryPatcher(_owmlConfig, _writer);
```

Comment:
I'd like this to be newed up either outside of this class and injected into the constructor, or newed up in the constructor :)

Reply:
👍